### PR TITLE
fix(filter-panel-accordion-item): use outline and transparent border to prevent hover + focus overlap

### LINKED ISSUE #373: Filter Panel - Accordion Item Active/Hover states
## Bug

When a `FilterPanelAccordionItem` is in an active state, and you hover over the `FilterPanelAccordionItem` below it, the hover state sits over top of the bottom border of the active state.

**Expected behavior -**

There should be no overlap.

**Actual behavior -**

There is overlap

### Steps for reproducing

Click a `FilterPanelAccordionItem` and hover over the the one below it.

### Screenshots

image

### Affected browsers

Tested on Chrome, assume all are impacted

### PR DESCRIPTION
**Pull request by jendowns**
## Affected issues

- Resolves #373

## Proposed changes

- use outline and a transparent border to prevent `FilterPanelAccordionItem`'s hover state from overlapping the focus outline of an adjacent `FilterPanelAccordionItem`

### CODE REVIEW

**Comment by jendowns** [comment]
@SimonFinney Would you have a chance to review this? Small bug fix. Would be nice to get @carbon-design-system/ibm-security-design to review as well.

File: src/components/FilterPanel/FilterPanelAccordionItem/_mixins.scss [hunk 1]
```diff
@@ -44,13 +44,21 @@
...
       // accordion heading label.
       &::before {
         left: -$carbon--spacing-01;
         width: calc(100% + #{$carbon--spacing-03});
       }
+
+      &:focus::before {
+        // Override Carbon `Accordion` defaults:
+        border: none;
+        outline: 2px solid $focus;
```
Comment (SimonFinney) [inline src/components/FilterPanel/FilterPanelAccordionItem/_mixins.scss]: Is it possible to use the `focus-outline` mixin here instead?

**Comment by jendowns** [comment]
Thanks @SimonFinney! Just applied the `focus-outline` mixin per your feedback

**Pull request merged**